### PR TITLE
Misc: fix logger.err, add catch and minor startup logging

### PR DESCRIPTION
**Pull request by qfox**
Without this fix it crashes silently.

### CODE REVIEW

**Comment by tadatuta** [comment]
cc @tormozz48 @gela-d

**Comment by gela-d** [comment]
:+1:

**Comment by gela-d** [comment]
@tavriaforever

File: src/blocks/server.blocks/app/app.node.js [hunk 1]
```diff
@@ -71,6 +75,9 @@ modules.define('app', ['config', 'logger', 'util', 'model', 'middleware', 'updat
                             updater.init();
                             updater.start();
                         }
+                    })
+                    .catch(function (e) {
```
Comment (tavriaforever) [inline src/blocks/server.blocks/app/app.node.js]: It is ok, but more coolest when using more usual `fail`, but it ok too.

**Comment by tavriaforever** [comment]
My soul is satisfied :sunny:

**Comment by qfox** [comment]
@tavriaforever replaced `.catch` with `.fail` ;-)

**Pull request merged**